pages/gm/f21.js

```diff
@@ -84,16 +110,74 @@ function F21() {
 				}}
 			/>
 			<Banner decorative />
+      <div className="text-center">
+        <img className="gm-graphic" src="/images/Fall_GM_2021_graphic.png" width="1000px" alt="Fall GM 2021 Marketing Graphic"/>
+      </div>
+      <Countdown date={GM_START_TIME} renderer={countdownRenderer}/>
 			<div className="content-container-tight text-center">
-				<h1>Fall &apos;21 General Meeting</h1>
-        <Link href={RSVP_LINK}>
-          <a className="button" target="_blank" rel="noreferrer noopener">
-            RSVP Now!
-          </a>
-        </Link>
-        <p>Check back later for more information on COVID safety, location, and the agenda!!</p>
+        <div id="info-wrapper">
+          <h2>Relevant information</h2>
+          <div className="flex">
+            <div>
+              <h3>How to get there</h3>
+              <img src="/images/Fall_GM_2021_temp_map.png" alt="temporary FPO image of google maps" width="400px"/>
+            </div>
+            <div className="what-to-bring">
+              <h3>What to bring</h3>
+              <p>Required: Face Mask, Bruincard</p>
+              <span>(learn more about how ACM is <Link href={"/covid"}><a target="_blank" rel="noreferrer noopener">committed to running COVID safe events</a></Link> in our return to campus)</span>
+              <p>Encouraged: Laptop or Phone to view slides, excitement to learn about ACM!</p>
+            </div>
+          </div>
+        </div>
+
+        <div id="gm-program-wrapper">
+          <h2>Program</h2>
+					<div className="gm-program-section">
+            <div className="section-header">
+              <h3>Welcome</h3>
+              <h4>[COPY CHECK] Welcome from president, Matt Wang</h4>
```

Comment:
If you have a chance, this should probably be a different tag (and maybe style with a class instead?)

Reply:
I'm using `h4s` here to take advantage of the poppins font, otherwise would have to do the styling myself.

If you want I can just do that with a div?

Reply:
Poppins font is easily doable from a SCSS file (w/ variables) as

```scss
.your-class {
  font-family: $font-header;
  // other stuff goes here
}
```

Hierarchy here is a bit weird since the `<h4>` isn't describing any heading. Part of a larger accessibility / semantic HTML push that I'd like to see, but obv constrained resources.